### PR TITLE
Issue - 2683 Default shorturls_length in the database if its not already set

### LINKED ISSUE #2683: Random ShortURLs plugin doesn't show default value
Upon first activation it should show the default value instead of a blank input:
image

(Also, not sure the HTML control is what we need here, a simple input text would be simpler. Was I the one to come up with this idea in the original plugin? Don't remember)

### PR DESCRIPTION
**Pull request by zach2825**
Also in the gitignore file, I added an entry for the idea directory

image

Closes #2683

### CODE REVIEW

File: user/plugins/random-shorturls/plugin.php [hunk 1]
```diff
@@ -61,7 +61,13 @@ function ozh_random_shorturl_settings_page() {
     }
 
     $random_length = yourls_get_option('random_shorturls_length');
-    $nonce = yourls_create_nonce( 'random_shorturl_settings' );
+    $nonce         = yourls_create_nonce('random_shorturl_settings');
```
Comment (dgw) [inline user/plugins/random-shorturls/plugin.php:64]: Isn't this also auto-formatting?

Comment (zach2825) [reply user/plugins/random-shorturls/plugin.php]: Thank you, keeps my honest. I did a few tests to make sure my ide will not auto-format anymore.

**Review by dgw — APPROVED** [review]
LGTM, assuming it matches @ozh's intent in #2683. 😸

Merger: don't forget to squash!

**Comment by ozh** [comment]
No need for the `if` block : `yourls_get_option()` accepts a default parameter when the value isn't set, see line 37 of the plugin. Less code, and saves a DB query storing the value.

**Comment by zach2825** [comment]
Then the value would only show on the form it does not mean it would exist in the database.

**Comment by zach2825** [comment]
As it's written now would cause a bug.

**Comment by ozh** [comment]
Why would it cause a bug? On the 2 occurrences of `yourls_get_option()`, we now have a default value.

"Fetching a value, and if not exists declare the value is 5" is equivalent to "Fetching a value, if not exists declare it is 5 and store it". You just skip the "store it" step.

**Comment by zach2825** [comment]
You're right, in YOURLS it's required to get options using the get_option method. I was thinking if people get the option directly from the database, which is a non-scenario.

Thank you

**Pull request merged**